activerecord/test/cases/relation/where_test.rb

```diff
@@ -359,6 +359,14 @@ def test_where_with_integer_for_binary_column
       assert_equal 0, count
     end
 
+    def test_where_with_sql_literal
+      subquery = Arel.sql("select id from posts where title like 'Welcome%'")
+      count = Post.where(id: subquery).count
```

Comment:
Is there any difference in the result if `subquery` is not wrapped in `Arel.sql`?

Reply:
Yes, then it is quoted as a string value and not a subquery.

Reply:
@rafaelfranca 

`WHERE id = "select id"` vs. `WHERE id = (select id)` ...is this what you were asking?

Reply:
Yes